testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -2424,6 +2424,14 @@ private static void checkConcatFunc(SqlOperatorFixture f) {
     f.checkFails("^concat()^", INVALID_ARGUMENTS_NUMBER, false);
   }
 
+  /** Test case for <a href="https://issues.apache.org/jira/browse/CALCITE-6518">
+   * ClassCastException during validation when loading multiple libraries</a>. */
+  @Test void issue6518() {
```

Comment:
I'm not sure issue6518 is appropriate, as there doesn't seem to be a similar test starting with that issue yet

Reply:
I will fix these and squash the commits at the same time, since this is a small PR. Hopefully we can merge it.